README.md

```diff
@@ -113,7 +140,10 @@ Your Answer:
 
 Select all that apply:
 ```
-[] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
+
+[X] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
 [] Synchronous code runs all at the same time, asynchronous code runs completely randomly
 [] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
+
+<!-- As I understand it, asynchronous code still runs in order, but the browser doesn't have to wait for that bit of code to finish running before it runs the next bit. Is this right? -->
 ```
```

Comment:
Exactly, I like to think of synchronous code as executing line-by-line, and therefore is blocking as it will not run the next line until the last line has been completed. In comparison, asych code is non-blocking, so the rest of the program will run without waiting for the block of code to execute.